### DISCUSSION TITLE
Audio Non Functional

### DISCUSSION THREAD

**Discussion by Willtheham** [guysoft/FullPageOS#553] [Q&A] [answered]
#### What were you doing?

Trying to turn volume up after plugging in AUX

#### What did you expect to happen?

Speakers working after plugging in the aux.

#### What happened instead?

No AUX audio output, tried ASAmixer, nothing, audio is on in config

#### Was there an error message displayed? What did it say?
no

#### Version of FullPageOS?
nightly

**Comment by guysoft** [comment]
Hey, the default audio interface is the HDMI, since you are plugging the Pi to the screen.
You can try to set it manually using rpi-config, that should persist between boots:
https://raspberrypi.stackexchange.com/a/61304/7483

This is not an issue so converting to a discussion.

**Reply by Willtheham** [reply]
I get "no audio systems known"

**Reply by Willtheham** [reply, marked as answer]
Installed Pulseaudio it then worked

**Reply by guysoft** [reply]
```
apt-get install pulseaudio 
```
?
I guess I could ship that. Or perhaps pipewire which means it would also have jack support.

**Reply by Willtheham** [reply]
Pulseaudio had jack support, right when it installed everything worked lol

**Reply by Willtheham** [reply]
But yes thats the command for it. just make sure it -r so it fully works.

**Reply by CrabbyPete** [reply]
It did not work for me. Still get 
Playing WAVE 'sample-6s.wav' : Signed 16 bit Little Endian, Rate 44100 Hz, Stereo
aplay: set_params:1352: Sample format non available
Available formats:
- IEC958_SUBFRAME_LE 

when I run aplay sample-6s.wav

**Reply by guysoft** [reply]
@CrabbyPete I see you posted it here to which looks like its an issue with that codec: https://github.com/RPi-Distro/pi-gen/issues/742

**Comment by guysoft** [comment]
Added pulseaudio to build so it will be shipped
